packages/server/api/routes/movies.router.js

```diff
@@ -14,6 +14,8 @@ router.get('/', (req, res, next) => {
     .then((result) => res.json(result))
     .catch(next);
 });
+// GET /featured-movie
+router.get('/featured-movie', moviesController.getFeaturedMovie);
```

Comment:
You could consider leaving the api path as /featured instead of /featured-movie

Reply:
You are right i was thinking what i should write but this is really simple and useful solution :)